web/src/features/service-worker/UpdatePrompt.tsx

```diff
@@ -12,17 +12,22 @@ function UpdatePrompt() {
     updateServiceWorker,
   } = useRegisterSW({
     onRegisteredSW: (_swURL, registration) => {
-      registration &&
-        setInterval(
-          () => {
-            console.info(`Checking for app update...`);
-            registration.update();
-          },
-          import.meta.env.PROD ? ONE_HOUR : 10 * 1000
-        );
+      if (registration) {
+        const checkForUpdates = async () => {
+          console.info('Checking for app update...');
+          try {
+            await registration.update();
+          } catch (error) {
+            console.warn('Failed to check for updates:', error);
+          }
```

Comment:
Nice addition!

Reply:
Hopefully this gives some more insight to our sentry errors.